pppd/main.c

```diff
@@ -1331,9 +1331,9 @@ print_link_stats(void)
 void
 reset_link_stats(int u)
 {
-    if (!get_ppp_stats(u, &old_link_stats))
-	return;
+    get_ppp_stats(u, &old_link_stats);
```

Comment:
Should we initialize old_link_stats to all zeroes if the get_ppp_stats call fails? Or copy it from link_stats?